src/screens/profile/components/navbar/styles.ts

```diff
@@ -0,0 +1,43 @@
+import { Dimensions, StyleSheet } from 'react-native';
+
+import { AppColor } from 'src/common/enums';
+
+const WIDTH = Dimensions.get('screen').width * 0.8;
+const BUTTON_WIDTH = Math.ceil(WIDTH / 3);
```

Comment:
You also use them in component, so extract to `src/constants` and add more meaningful name